### ISSUE TITLE
make option -c of command osm2psql adjustable

### ISSUE THREAD

**Issue by stevo01** [Overv/openstreetmap-tile-server#65]
Adjustment of parameter "-c" for command osm2psql is missing. I would like to add environment variable "NCACHE" to allow the adjustment of the cache size. 

I used and verified following patches to adjust parameter "-c":
```
diff --git a/Dockerfile b/Dockerfile
index 3b091b5..bc85c16 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -7,6 +7,7 @@ FROM ubuntu:18.04
 ENV TZ=UTC
 ENV AUTOVACUUM=on
 ENV UPDATES=disabled
+ENV NCACHE=2048
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
 # Install dependencies
```
```
diff --git a/run.sh b/run.sh
index a5e90ce..b722705 100755
--- a/run.sh
+++ b/run.sh
@@ -54,7 +54,7 @@ if [ "$1" = "import" ]; then
     sudo chown renderer:renderer /flat
 
     # Import data
-    sudo -u renderer osm2pgsql -d gis --create --slim -G --hstore --tag-transform-script /home/renderer/src/openstreetmap-carto/openstreetmap-carto.lua -C 2048 --number-processes ${THREADS:-4} ${OSM2PGSQL_EXTRA_ARGS} -S /home/renderer/src/openstreetmap-carto/openstreetmap-carto.style /data.osm.pbf
+    sudo -u renderer osm2pgsql -d gis --create --slim -G --hstore --tag-transform-script /home/renderer/src/openstreetmap-carto/openstreetmap-carto.lua -C ${NCACHE:-2048} --number-processes ${THREADS:-4} ${OSM2PGSQL_EXTRA_ARGS} -S /home/renderer/src/openstreetmap-carto/openstreetmap-carto.style /data.osm.pbf
 
     # Create indexes
     sudo -u postgres psql -d gis -f indexes.sql
```

Alternative solution: remove the "-c 2048" from run.sh script. This modification allows user to change the default value with docker environment variable "OSM2PGSQL_EXTRA_ARGS".

**Comment by Overv** [comment]
I like the solution of handling this from the extra args. I'll close this once that patch has been merged.

**Comment by stevo01** [comment]
issue is solved with a2eceb6bb50590186673537cc05fe6099708d2f2
